docs/deploy/rbac/rbac.yaml

```diff
@@ -14,7 +14,7 @@ rules:
     resources: ["certificates", "issuers", "clusterissuers"]
     verbs: ["*"]
   - apiGroups: [""]
-    resources: ["secrets", "events", "endpoints", "services", "pods"]
```

Comment:
These values should not be changed manually, and instead should be changed in the helm chart template and then re-generated with `./hack/update-deploy-gen.sh`.

Reply:
Ok, I went and ran that script - but the automatically generated change is precisely the same as my one?

Or do you mean I shouldn't commit the changes here, and just leave it at the helm chart changes?

Reply:
Oh ignore me - I thought you hadn't updated the Helm chart as part of this PR, but you have :smile: